Breeding-Insight/bi-web
# [BI-1366] subscribe ontology: bug fix for unsubscribe ontology

# Description
**Story:** https://breedinginsight.atlassian.net/jira/software/c/projects/BI/boards/1?modal=detail&selectedIssue=BI-1366

After subscribing to an ontology, the traits check would return traits and so a message would display:

```This program contains traits and therefore cannot subscribe to another program's ontology.```

Added a condition in that check to see if the program was already subscribed to an ontology. 


# Dependencies
biapi:release/0.6

# Testing
1. Create a program with traits
2. Create a program without traits
3. Share Traits Program with NoTraits Program
4. Subscribe NoTraits Program to Traits Program
5. Check that the 'Unsubscribe' option is shown after subscribing. 


# Checklist:
